gotocompany/compass/v1beta1/service.proto

```diff
@@ -810,6 +810,21 @@ message SearchAssetsRequest {
       description: "offset parameter defines the offset from the first result you want to fetch"
     }
   ];
+
+  SearchFlags flags = 8 [ (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "add flags based on which query parameters can be changed."
```

Comment:
Can you change it to something like - `search flags to control the search behavior`

Reply:
Done